### PR TITLE
feat(storage-info): Add storage info endpoint to the admin API client

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

**Comment by ThetaSinner** [comment]
The test isn't happy because `storage_info` doesn't exist in the `holochain` currently pointed to by holonix - still working out what to do about that

**Comment by jost-s** [comment]
Until we have a holonix version for 0.2 we can only update the flake to reference `develop`. I don't want to do that.
I'll run the tests locally too and then we should be good. It's not a potential source of a lot of bugs =)

File: src/api/admin/types.ts [hunk 1]
```diff
@@ -706,6 +706,43 @@ export interface FullStateDump {
...
+  cache_data_size: number;
+  cache_data_size_on_disk: number;
+  used_by: Array<InstalledAppId>;
+}
+
+/**
+ * @public
+ */
+export interface DnaStorageBlob {
+  Dna: DnaStorageInfo;
```
Comment (jost-s) [inline src/api/admin/types.ts:726]:
```suggestion
  dna: DnaStorageInfo;
```

**Review by jost-s — APPROVED** [review]
Great!

Tested locally with Holochain v0.2.0-beta-rc.4 with success.

It's quite late where you are, so I lower-case the property and merge and release this.

File: src/api/admin/types.ts (hunk 1 above)
Comment (jost-s) [reply src/api/admin/types.ts:726]: This has to be fixed upstream in Holochain. The Conductor API should produce all lower case snake case fields/properties.

Comment (jost-s) [reply src/api/admin/types.ts:726]: fixed by https://github.com/holochain/holochain/pull/2234

**Comment by jost-s** [comment]
> Until we have a holonix version for 0.2 we can only update the flake to reference `develop`. I don't want to do that. I'll run the tests locally too and then we should be good. It's not a potential source of a lot of bugs =)

Actually I pointed the flake to rc.4 after all.

**Pull request merged**